src/2_mockFn.test.js

```diff
@@ -1,3 +1,17 @@
-// Your tests here...
-// You'll practice the basics of jest mocks
-it("...", () => {});
+import { guardMove } from "./2_mockFn";
+
+describe("Moving a Guard", () => {
+  test("Sends a move or run action to the guard", () => {
+    const realFuntion = (value) => {
+      if (["walk", "run"].includes(value)) {
+        return value;
+      }
+    };
+
+    const shadowCallback = jest.fn(realFuntion);
+
+    const result = guardMove(shadowCallback, "walk", 7);
+    expect(shadowCallback).toHaveBeenCalledTimes(1);
```

Comment:
You forgot to assert what the `shadowCallback` was called with. in this case, it was called with `walk` and `7`